Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -1053,7 +1055,6 @@ void HTMLInputElement::create_range_input_shadow_tree()
         auto rect = get_bounding_client_rect();
         double minimum = *min();
         double maximum = *max();
-        // FIXME: Snap new value to input steps
```

Comment:
Is this fixme resolved?

Reply:
I thought it was but after some testing I found some bugs still, so I keep it there